test/ffmpeg-qsv/encode/hevc.py

```diff
@@ -60,6 +60,26 @@ def test_r2r(self, case, gop, slices, qp, quality, profile):
     vars(self).setdefault("r2r", 5)
     self.encode()
 
+class cqp_seek(HEVC8EncoderTest):
+  def init(self, tspec, case, gop, slices, bframes, qp, quality, profile, seek):
+    vars(self).update(tspec[case].copy())
+    vars(self).update(
+      bframes = bframes,
+      case    = case,
+      gop     = gop,
+      qp      = qp,
+      quality = quality,
+      profile = profile,
+      rcmode  = "cqp",
+      slices  = slices,
+      seek    = seek,
+    )
+
+  @slash.parametrize(*gen_hevc_cqp_seek_parameters(spec, ['main']))
+  def test(self, case, gop, slices, bframes, qp, quality, profile, seek):
+    self.init(spec, case, gop, slices, bframes, qp, quality, profile, seek)
+    self.encode()
+
```

Comment:
Maybe it would be better to use similar test signature like the `pict` test classes.  This way, we would not need different classes for cqp, cbr, tcbrc, and vbr variations.  That is, the rcmode can be passed as a parameter and perhaps integrate tcbrc params into one test definition.